sql/2022/markup/anchors_schemes.sql

```diff
@@ -0,0 +1,30 @@
+WITH totals AS (
+  SELECT
+    _TABLE_SUFFIX,
+    COUNT(0) AS total
+  FROM
+    `httparchive.summary_pages.2022_06_01_*`
+  GROUP BY
+    _TABLE_SUFFIX
+)
+
+SELECT
+  _TABLE_SUFFIX AS client,
+  REGEXP_EXTRACT(href, r'^[\'"]?(\w+):') AS scheme,
```

Comment:
Not sure what `hrefs_without_special_scheme` means but any risk this could look at relative URLs?

Cause [apparently a colon os allowed in certain parts of a URL](https://stackoverflow.com/questions/2053132/is-a-colon-safe-for-friendly-url-use) so could match here if that does allow relative URLs.